cupy_backends/cuda/libs/cublas.pyx

```diff
@@ -1029,14 +1038,24 @@ cpdef gemmEx(
         int ldc, int computeType, int algo):
     _setStream(handle)
     with nogil:
-        status = cublasGemmEx(
-            <Handle>handle, <Operation>transa, <Operation>transb, m, n, k,
-            <const void*>alpha,
-            <const void*>A, <runtime.DataType>Atype, lda,
-            <const void*>B, <runtime.DataType>Btype, ldb,
-            <const void*>beta,
-            <void*>C, <runtime.DataType>Ctype, ldc,
-            <runtime.DataType>computeType, <GemmAlgo>algo)
+        if computeType >= CUBLAS_COMPUTE_16F:
+            status = cublasGemmEx_v11(
```

Comment:
Question: It seems there's a C++ overloaded version of `cublasGemmEx` that supports the old `cudaDataType`?https://docs.nvidia.com/cuda/cublas/index.html#cublas-GemmEx
I wonder if using that could help, or is it to be deprecated soon?

Reply:
That's right, even with `cublasGemmEx` of CUDA11, you can still specify the compute type with `cudaDataType`, as long as you're in C++. However, in the old way, you cannot specify, for example, TF32 (TensorFloat32) as a compute type, so you need to specify the compute type with `cublasComputeType`, which is added in CUDA11.

Reply:
Ah I see, so this is why we need the new interface...